sources/packages/web/src/components/common/students/StudentApplicationsSimplifiedSummary.vue

```diff
@@ -0,0 +1,162 @@
+<!-- This component is shared between ministry and institution users. -->
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header
+        title="Applications"
+        :recordsCount="applicationsAndCount.count"
+      ></body-header>
+    </template>
+    <content-group>
+      <toggle-content
+        :toggled="!applicationsAndCount.results?.length"
```

Comment:
May or may not be part of PR, but toggled must be based on `applicationsAndCount.count`

Reply:
I will adjust either way.

Reply:
Adjusted on both components.